### PR TITLE
Rename RCT to AE and change package name

### PR DESCRIPTION
**Pull request by dvicory**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,17 +54,17 @@ pod 'react-native-camera', path: '../node_modules/react-native-camera'
...
+5. Click `AECamera.xcodeproj` in the project navigator and go the `Build Settings` tab. Make sure 'All' is toggled on (instead of 'Basic'). In the `Search Paths` section, look for `Header Search Paths` and make sure it contains both `$(SRCROOT)/../../react-native/React` and `$(SRCROOT)/../../../React` - mark both as `recursive`.
 5. Run your project (`Cmd+R`)
 
 
 #### Android
 1. `npm install react-native-camera@https://github.com/lwansbrough/react-native-camera.git --save`
 2. Open up `android/app/src/main/java/[...]/MainApplication.java
-  - Add `import com.lwansbrough.RCTCamera.RCTCameraPackage;` to the imports at the top of the file
-  - Add `new RCTCameraPackage()` to the list returned by the `getPackages()` method. Add a comma to the previous item if there's already something there.
+  - Add `import com.appfolio.AECamera.AECameraPackage;` to the imports at the top of the file
```
Comment (jonkessler) [inline README.md]: `com.appfolio.react.camera.CameraPackage` ?

File: README.md [hunk 2]
```diff
@@ -54,17 +54,17 @@ pod 'react-native-camera', path: '../node_modules/react-native-camera'
...
+  - Add `new AECameraPackage()` to the list returned by the `getPackages()` method. Add a comma to the previous item if there's already something there.
```
Comment (jonkessler) [inline README.md]: `new CameraPackage()`?

File: ios/AECamera.h [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#import <UIKit/UIKit.h>
+#import <AVFoundation/AVFoundation.h>
+#import "CameraFocusSquare.h"
+
+@class AECameraManager;
+
+@interface AECamera : UIView
+
+- (id)initWithManager:(AECameraManager*)manager bridge:(RCTBridge *)bridge;
```
Comment (jonkessler) [inline ios/AECamera.h]: Need a space between `AECameraManager` and `*`.

File: ios/AECamera.m [hunk 4]
```diff
@@ -70,7 +70,7 @@ - (void)setOnZoomChanged:(BOOL)enabled
   }
 }
 
-- (id)initWithManager:(RCTCameraManager*)manager bridge:(RCTBridge *)bridge
+- (id)initWithManager:(AECameraManager*)manager bridge:(RCTBridge *)bridge
```
Comment (jonkessler) [inline ios/AECamera.m]: Missing space.

**Comment by dvicory** [comment]
[[Android] App usually crashes on launch](https://trello.com/c/syAvD39p/249-android-app-usually-crashes-on-launch)

**Review by jonkessler — CHANGES_REQUESTED** [review]
Search for "rctcamera" case-insensitive in the project.. there are still some references.

**Review by jonkessler — APPROVED** [review]

**Pull request merged**